youtube_dl/extractor/common.py

```diff
@@ -2167,7 +2167,16 @@ def _find_jwplayer_data(webpage):
             r'jwplayer\((?P<quote>[\'"])[^\'" ]+(?P=quote)\)\.setup\s*\((?P<options>[^)]+)\)',
             webpage)
         if mobj:
-            return mobj.group('options')
+            options = mobj.group('options').strip()
+            # We're looking for JSON
+            if options[0] != '{':
+                # setup() got a variable as an argument - resolve it
+                regex = r'%s\s*=\s*(?P<options>\{[^\}]\})' % re.escape(options)
+                mobj = re.search(regex, webpage)
```

Comment:
Sorry for previous noises. I just actually checked the HTML source of your example and found that this PR doesn't work as expected. ("resolving the variable to the JSON string") Try to print the value of ```mobj``` after this line.